<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>agents</class>
 <widget class="QDialog" name="agents">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/splashBG.jpg);</string>
  </property>
  <widget class="QLabel" name="BG">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>750</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pics/splashBG.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_agents">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/pics/agentsTitle.png) 0 0 0 0 stretch stretch; ;
background:transparent;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_hunters">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>-10</y>
     <width>371</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/hunters.PNG) 0 0 0 0 stretch stretch;
background:transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_preys">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>60</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-image: url(:/images/prey.PNG) 0 0 0 0 stretch stretch;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_h">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>480</width>
     <height>580</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px dotted white;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QScrollArea" name="scrollArea_h">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px dotted white;</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>457</width>
       <height>1200</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1200</height>
      </size>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>461</width>
        <height>1204</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton_h13">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="pushButton_h20">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_h10">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButton_h14">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="pushButton_h15">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_h01">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="pushButton_h21">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="pushButton_h17">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="pushButton_h22">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pushButton_h19">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="pushButton_h16">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_h09">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_h07">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_h05">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_h03">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_h04">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="pushButton_h23">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_h02">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_h08">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QPushButton" name="pushButton_h24">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="pushButton_h18">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_h06">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton_h12">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_h11">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_p">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>480</width>
     <height>580</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px dotted white;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px dotted white;</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>457</width>
       <height>1150</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1150</height>
      </size>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>461</width>
        <height>1171</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="7" column="0">
        <widget class="QPushButton" name="pushButton_p22">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QPushButton" name="pushButton_p24">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_p10">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="pushButton_p18">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="pushButton_p17">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="pushButton_p23">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_p07">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_p06">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_p09">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton_p12">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="pushButton_p15">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pushButton_p19">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_p01">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_p03">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="pushButton_p21">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="pushButton_p20">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_p08">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_p02">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButton_p14">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_p05">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_p04">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_p11">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton_p13">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="pushButton_p16">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_moreInfo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>700</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px dotted white;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_picture">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>300</width>
      <height>310</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:transparent;
background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>481</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:transparent;
background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_HP">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>260</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:transparent;
background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Attack">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>260</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:transparent;
background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Range">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>370</y>
      <width>260</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:transparent;
background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Mobility">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>290</y>
      <width>260</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:transparent;
background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/images/icons/undo.png);
border:transparent;
background:transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_previous">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>450</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:transparent;
background:transparent;

image: url(:/images/icons/previous.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>450</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/images/icons/next.png);
border:transparent;
background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_pictureHP">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:transparent;
background:transparent;

image: url(:/images/icons/health.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_pictureAttack">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:transparent;
background:transparent;
image: url(:/images/icons/attack.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_pictureMobility">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:transparent;
background:transparent;

image: url(:/images/icons/mobility.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_pictureRange">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:transparent;
background:transparent;

image: url(:/images/icons/range.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_type">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:transparent;
background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
